backend/tests/unit/test_crud.py

```diff
@@ -0,0 +1,380 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from app.main import crud, models, schemas
+from app.main.database import Base
+from tests.api.base_api_test import BasicApiTestCase
+
+SQLALCHEMY_DATABASE_URL = "postgresql://mealadmin:happymeal@localhost/mealcare_test"
```

Comment:
Can we take advantage of @atykwonderland 's work with the .env file?

Reply:
@Ty-Won  I can't seem to find the work, can you link .env file?

Reply:
@taerim25 .env file is something we don't commit since we store sensitive info in there so theres no link it's something you create yourself :). This is a local test db so not the biggest deal but it's good to keep habits.

Reply:
Please look into creating a .env file that is something hidden from git. Something similar is written in here: https://fastapi.tiangolo.com/advanced/settings/#read-env-vars-in-python except they just export the variable through the command line (You could do, but I suggest creating a file). To create an environment file, you would create a hidden .env file and put it in the gitignore. Once done, your code will have to target that file to extract variables in there. Check this out: https://fastapi.tiangolo.com/advanced/settings/#reading-a-env-file